docs/index.md

```diff
@@ -34,28 +34,52 @@ Be sure to install [Vue Syntax Highlight](https://packagecontrol.io/packages/Vue
 
 ### Python<a name="python"></a>
 
-`pip install python-language-server`
+There are at least two language servers, use either one.
 
-See: [github:palantir/python-language-server](https://github.com/palantir/python-language-server)
+#### Palantir Python Language Server
 
-Alternatively, Microsoft's python language server (using .NET Core runtime)
+```sh
+pip install python-language-server
+```
 
-[Instructions here](https://github.com/Microsoft/python-language-server/blob/master/Using_in_sublime_text.md)
+Make sure you can run `pyls` in your terminal. If you've installed it into a virtualenv, you might need to override the path to `pyls` in global LSP settings (Package Settings -> LSP -> Settings):
 
-Use virtualenv adding the following settings :
 
 ```json
-"settings": {
+{
+    "clients": {
+        "pyls": {
+            "command": ["/Users/mike/.virtualenvs/pyls-virtual-env/bin/pyls"]
```

Comment:
Perhaps you can mention that `Users/mike` should be replaced by the path to the reader's `pyls`. For an example of that, see the `dart` configuration.
You might also want to mention to set `"enabled": true` here because there have been users in the past that got confused about that boolean setting.